ui/apps/platform/src/Containers/Vulnerabilities/PlatformCves/PlatformCve/ClustersByTypeSummaryCard.tsx

```diff
@@ -23,32 +23,43 @@ export type ClustersByType = {
 };
 
 export type ClustersByTypeSummaryCardProps = {
-    clusterCounts: ClustersByType['clusterCountByType'];
+    clusterCounts?: ClustersByType['clusterCountByType'];
 };
 
 function ClustersByTypeSummaryCard({ clusterCounts }: ClustersByTypeSummaryCardProps) {
-    const { generic, kubernetes, openshift, openshift4 } = clusterCounts;
+    const counts = clusterCounts ?? { generic: 0, kubernetes: 0, openshift: 0, openshift4: 0 };
+    const { generic, kubernetes, openshift, openshift4 } = counts;
+    const totalCount = generic + kubernetes + openshift + openshift4;
```

Comment:
What do you think about this modification? 

```
const { generic = 0, kubernetes = 0, openshift = 0, openshift4 = 0 } = clusterCounts ?? {};
const totalCount = generic + kubernetes + openshift + openshift4;
```

Reply:
Nice! I like it.